src/utils/viewer/viewer.cpp

```diff
@@ -1088,27 +1124,19 @@ void Viewer::drawEvent() {
   if (depthMode_) {
     // ================ Depth Visualization ==================================
     std::string sensorId = "depth";
-    if (fisheyeMode_) {
+    if (sensorMode_ == (uint8_t)VisualSensorMode::Fisheye) {
```

Comment:
NIT: Avoid C Style Casts

Reply:
I'm usually doing

```suggestion
    if (sensorMode_ == uint8_t(VisualSensorMode::Fisheye)) {
```

Which *technically* is a C-style cast as well I think(?), but `-Wold-style-cast` doesn't complain and it's short enough to not be annoying. Only using it for builtin types though.

Reply:
On the other hand, I'm wondering why not just have `sensorMode_` as `VisualSensorMode` and do the casting on the *single* place where `+` and `%` is used on it, instead of casting all other cases? :)